javascript/features/death_match/death_match_manager.js

```diff
@@ -214,6 +214,10 @@ export class DeathMatchManger {
         wait(1).then(() => player.virtualWorld = location.world);
         player.interiorId = location.interiorId;
 
+        // Force-update the streamer for the player, based on where they will be spawning.
+        player.updateStreamer(spawnPosition.position, location.world,
+        location.interiorId, /* STREAMER_TYPE_OBJECT= */ 0);
```

Comment:
NIT: You'll want to add an extra tab (indenting) here

Reply:
it's not working :(

Reply:
Callback (98) ignored because a ScopedIgnore is in place.
[23:10:38] [WARNING][callback_hook.cc:80] Callback (OnPlayerActivityChange) ignored << because a ScopedIgnore is in place.
[23:10:38] [WARNING][callback_hook.cc:80] Callback (OnPlayerActivityChange) ignored << because a ScopedIgnore is in place.

Reply:
How about you restrict a player from using the command when they're in a car? :)

Reply:
yeah. I think that's the only way now.